sdk/tables/Azure.Data.Tables/samples/README.md

```diff
@@ -0,0 +1,28 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-tables
+name: Azure Tables samples for .NET
+description: Samples for the Azure.Data.Tables client library
+---
+
+# Azure Tables client SDK Samples
+Description of Azure Tables. Covers following functions: 
+* Create and delete a table
+  * Includes error handling
+* Lists or queries tables
+* Inserts and deletes entities
+  * Includes error handling
+* Update/upsert entity data using optimistic concurrency checks
+* Create odata query/listing entities
+* Build and commit a batch of operations
+
+You can find samples for each of this main functions below, as well as a sample on how to create a mock client for testing purposes.
+To get started you'll need an Azure Tables endpoint and credentials. See Azure Tables Client Library [Readme][README] for more information and instructions.
```

Comment:
I believe the Readme link here should point to the [main readme](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/readme.md), 

I suggest filling out the `Getting Started` and `Prerequisites` sections as they are pretty straight forward. You can grab the pre-req content from another storage service like [here](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Queues/README.md#prerequisites) for now, but at some point we'll need to add instructions for creating a cosmos table resource as well.

Reply:
To keep this PR short, I will suggest create an issue on Github to add those sections to the readme.
Later you can go and do that. Let me know if that makes sense